tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/WriteSourceRepoProperties.cs

```diff
@@ -175,16 +174,7 @@ private string GetCommitCount(string gitDir, string hash)
 
         private string GetCommitDate(string gitDir, string hash)
         {
-            string format;
-            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Windows)
-            {
-                format = "%%cd";
-            }
-            else
-            {
-                format = "%cd";
-            }
-            return RunGitCommand($"log -1 --format={format} --date=short {hash}", gitDir: gitDir);
+            return RunGitCommand($"log -1 --format=%cd --date=short {hash}", gitDir: gitDir);
```

Comment:
If we go back to supporting Windows, this will have to be added back?

Reply:
Kind of.  It would have to be messier since this can't rely on core-setup to tell it the platform anymore - we'd probably have to change this to a try-catch and attempt both ways.